Tests/Benchmarks/Benchmarks.csproj

```diff
@@ -16,5 +16,6 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Data.DataSetExtensions" Condition="'$(TargetFramework)' == 'net472'" />
+    <Reference Include="System.Net.Http" Condition="'$(TargetFramework)' == 'net472'" />
```

Comment:
Why is this needed?

Reply:
With the "new" .NET (.NET Standard, .NET Core, .NET ≥ 5.0), `System.Net.Http` has become part of .NET's _Base Class Libraries_. Therefore the assembly is automatically and implicitly referenced in all newer projectes.

But with .NET Framework, this is not the case. The assembly is always present in the GAC (because the .NET Framework installs it), but it has to be referenced manually.